app/components/Dashboard/BestNewMusicTab/BestNewMusicContent/index.js

```diff
@@ -17,34 +16,34 @@ const BestNewMusicContent = props => {
   if (item === null) {
     return null;
   }
-  
+
   return (
     <div className={styles.best_new_music_content}>
       <div className={styles.review_header}>
         <div className={styles.thumbnail}>
-          <img alt={ item.title } src={ item.thumbnail }/>
```

Comment:
My automatic code formatter integrated with ESLint keep removing those spaces. I can fix that, although I haven't found the appropriate setting for that.